src/org/labkey/test/util/TestDataGenerator.java

```diff
@@ -72,6 +77,13 @@ public class TestDataGenerator
     public static final String ALPHANUMERIC_STRING = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqrstuvxyz";
     public static final String DOMAIN_SPECIAL_STRING =  "+- _.:&()/";
     public static final String ILLEGAL_DOMAIN_NAME_CHARSET = "<>[]{};,`\"~!@#$%^*=|?\\";
+    // Used to set value of date pickers
+    public static final Supplier<SimpleDateFormat> INPUT_DATE_FORMAT = () -> new SimpleDateFormat("MM/dd/yyyy");
+    public static final Supplier<SimpleDateFormat> INPUT_DATETIME_FORMAT = () -> new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+    // Default display format
+    public static final Supplier<SimpleDateFormat> UI_DATE_FORMAT = () -> new SimpleDateFormat("yyyy-MM-dd");
+    public static final Supplier<SimpleDateFormat> UI_DATE_TIME_FORMAT = () -> new SimpleDateFormat("yyyy-MM-dd HH:mm");
+    public static final Supplier<SimpleDateFormat> TIME_FORMAT = () -> new SimpleDateFormat("HH:mm:ss");
```

Comment:
These formats are enums in `BaseSettingsPage.DATE_FORMAT`.
I don't know what the best solution is but I would like to see the test automation changed to have values like these formats, or various "tricky characters" string and system defined field names / labels (e.g. Sample ID, Sample ID *, etc...) in a more generalized resource file or class.

Obviously beyond the scope of this PR.

Reply:
Ah, shoot. Also missed this in my eagerness to merge. If these are available elsewhere, I can update to use the other values in my current branch (assuming I remember).